src/modules/Dashboard/features/Applet/Password/EnterAppletPassword/EnterAppletPassword.tsx

```diff
@@ -29,6 +29,12 @@ export const EnterAppletPassword = forwardRef<AppletPasswordRef, EnterAppletPass
     const [showPassword, setShowPassword] = useState(false);
 
     const submitForm = ({ appletPassword }: EnterAppletPasswordForm) => {
+      if (isApplet) {
+        submitCallback && submitCallback({ appletPassword });
+
+        return;
+      }
+
       const appletEncryption = encryption || getAppletData(appletsFoldersData, appletId).encryption;
```

Comment:
It's not clear in which cases should apply encryption logic.

Reply:
if the user inputs password to check it if it's corresponds to the password from the backend